component.json

```diff
@@ -61,7 +61,8 @@
     "segmentio/json": "^1.0.0",
     "component/reduce": "1.0.1",
     "ianstormtaylor/pick": "~0.1.0",
-    "harrietgrace/omit": "~0.1.0"
+    "harrietgrace/omit": "~0.1.0",
+    "darkskyapp/string-hash": "*"
```

Comment:
`*` is pretty much never what you want; is there a particular reason we're using it here? Otherwise pin it to a specific version